.gitignore

```diff
@@ -1,3 +1,6 @@
 scalalib/src/
 target/
+submodules/scala
 *.dot
+.idea
+.idea_modules
```

Comment:
This is environment-specific. It should be in your global `.gitignore`, not in project-specific `.gitignore`s.